server/core_authenticate.go

```diff
@@ -477,6 +492,16 @@ func AuthenticateGoogle(ctx context.Context, logger *zap.Logger, db *sql.DB, cli
 			return "", "", false, status.Error(codes.Unauthenticated, "Error finding or creating user account.")
 		}
 
+		// Check if display name / avatar received from Google different from those in the DB
+		if dbDisplayName.String != displayName || dbAvatarUrl.String != avatarUrl {
```

Comment:
Note for the team: We are now introducing a hit to the database on re-authentication (which we've never had before). I'm OK with this exception because I assume this will be an infrequent operation (i.e. users won't update their name/avatar frequently).

@novabyte @zyro Post your thoughts here, and if OK we can merge this.

Reply:
On the same boat as @mofirouz. Evaluating the situation in totality, downside of avoiding this database hit is showing an outdated name / profile pic to the user which would be worse, IMO.

Reply:
I'm OK with this tradeoff. Let's try and keep the info in sync - it should not change often enough to have an impact.